<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="newGameButton">
       <property name="text">
        <string>Новая игра</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gameMenuButton">
       <property name="text">
        <string>Меню игры</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recordsMenuButton">
       <property name="text">
        <string>Меню рекордов</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>119</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="gamePage">
     <widget class="QPushButton" name="checkButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>161</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputField">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>541</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="gameLog">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>64</y>
        <width>751</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="recordsPage">
     <widget class="QListWidget" name="recordsList">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>581</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
